### PR TITLE
docs: readme style and project status

### PR DESCRIPTION
**Pull request by tisonkun**
I hereby agree to the terms of the [GreptimeDB CLA](https://github.com/GreptimeTeam/.github/blob/main/CLA.md).

## What's changed and what's your intention?

1. Header style
2. Getting started
3. Project status

[Rendered](https://github.com/tisonkun/greptimedb/blob/more-readme/README.md)

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.
- [x]  This PR does not require documentation updates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,70 +6,105 @@
...
 
-```
-42
-```
-
-Read more on docs:
+## Getting Started
 
-* [Installation](https://docs.greptime.com/getting-started/installation/overview)
 * [Quickstart](https://docs.greptime.com/getting-started/quick-start/overview)
```
Comment (tisonkun) [inline README.md:108]: Potential - more starting points like [this style](https://github.com/datafuselabs/databend/blob/main/README.md#-getting-started)

File: README.md [hunk 2]
```diff
@@ -95,11 +130,6 @@ Run a standalone server:
 cargo run -- standalone start
```
Comment (tisonkun) [inline README.md:130]: Potential - move to CONTRIBUTING and revise that guide.

**Review by sunng87 — APPROVED** [review]
Awesome!

**Review by killme2008 — APPROVED** [review]
LGTM

**Review by waynexia — APPROVED** [review]

**Pull request merged**